tests/rptest/tests/delete_records_test.py

```diff
@@ -361,44 +356,41 @@ def test_delete_records_empty_or_missing_topic_or_partition(self):
         missing_topic = "foo_bar"
         out_of_range_prefix = "OFFSET_OUT_OF_RANGE"
         unknown_topic_or_partition = "UNKNOWN_TOPIC_OR_PARTITION"
-        missing_partition = "partition was not returned"
```

Comment:
I had to remove this since rpk outputs for an unknown topic and a missing partition is the same: `UNKNOWN_TOPIC_OR_PARTITION`

Cc: @graphcareful

Reply:
all good